tasks/create-product-sbom/create-product-sbom.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: create-product-sbom
+  labels:
+    app.kubernetes.io/version: "1.0.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Create product-level SBOM from release notes.
+  params:
+    - name: dataJsonPath
+      description: Relative path to the JSON data file in the workspace.
+      type: string
+    - name: subdirectory
```

Comment:
I know Martin said either a subdirectory parameter or another method, but I think it is much cleaner with the other method he suggested. Not only does it result in less parameters to pass, but it is more error proof as it will put the sbom in the same spot as every other resource for the PLR, and other tasks probably look only there

Reply:
addressed